python/cudf/cudf/tests/test_groupby.py

```diff
@@ -2086,3 +2086,21 @@ def test_groupby_describe(data, group):
     expect = pdf.groupby(group).describe()
 
     assert_groupby_results_equal(expect, got, check_dtype=False)
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        {"a": [], "b": []},
+        {"a": [2, 1, 2, 1, 1, 3], "b": [None, 1, 2, None, 2, None]},
+        {"a": [None], "b": [None]},
+        {"a": [2, 1, 1], "b": [None, 1, 0], "c": [None, 0, 1]},
+    ],
+)
+@pytest.mark.parametrize("agg", ["first", "last", ["first", "last"]])
+def test_groupby_first(data, agg):
+    pdf = pd.DataFrame(data)
+    gdf = cudf.from_pandas(pdf)
```

Comment:
Everything looks good to me! Would only add a test case for a dataframe with  empty and non-empty values.

Reply:
Thanks!